_migrations/migration-phases/verifying-tools-for-migration.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: default
+title: Verifying tools for migrations
+nav_order: 70
+parent: Migration phases
+---
+
+# Verifying tools for migration
+
+Before using the migration assistant, take the following steps to verify that you're cluster is ready for migration.
```

Comment:
Inconsistent casing. (As a side note, I know we use “Migration Assistant,” but since Migration Assistant is the name of a product, I think we can remove “the” in each of these occurrences. These changes should be applied consistently.)